### ISSUE TITLE
Allow showing Tags in Entry Subtitle

### ISSUE THREAD

**Issue by stefanschell** [keepassium/KeePassium#364]
I would like to have the option to show Tags as an Entry Subtitle. Currently, the Entry Subtitle can be configured to be all kinds of things, but not the Tags.

I assume if there are too many Tags, or if they are too long, they could be shortened with trailing "...", e.g. like "tag1, tag2, tag3, ...".

The wish above is one way of showing the Tags in the list of Entries. I guess what I ultimately need is a quick way to see all Entries with their associated Tags (I am aware of the filter functionality, but for this I need to know what Tags exist).

**Comment by keepassium** [comment]
Thank you, @stefanschell. This is easy to add, but first it would be useful to dig a bit into the "why".

>  I guess what I ultimately need is a quick way to see all Entries with their associated Tags (I am aware of the filter functionality, but for this I need to know what Tags exist).

When you start editing tags, there is a list "All tags":
![tag-selector](https://github.com/keepassium/KeePassium/assets/38399598/bbb8798c-427a-4530-b312-ab54867fd289)

Is it too buried to be useful, or it does not show associated entries, or something else should be improved?

**Comment by stefanschell** [comment]
Let me try again to formulate what I would like to solve.

I would like to see the Tags next to each Entry to help me remember quickly what a certain Entry actually is for. I usually have tags like: Work, Government, Shopping, or Technology. Without the Tags it is just the name of a service and that doesn't always ring a bell when looking at my list of Entries. I look over the list of entries to keep an overview about my current services. Maybe I could get rid of one, for example. Or maybe I have to change my phone number, which might be more urgent for an Entry with the Government Tag. 

Having to look up Tags by clicking on each entry is a bit too cumbersome. I guess I am using the Tags as an alternative to Groups, with the obvious advantage of being able to assign multiple ones.

**Comment by stefanschell** [comment]
Thanks.
